connector/pom.xml

```diff
@@ -132,8 +132,18 @@
                     <groupId>net.kyori</groupId>
                     <artifactId>adventure-text-serializer-gson</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>com.github.steveice10</groupId>
+                    <artifactId>packetlib</artifactId>
+                </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>com.github.steveice10</groupId>
+            <artifactId>PacketLib</artifactId>
+            <version>54f761c</version>
+            <scope>compile</scope>
+        </dependency>
         <dependency>
             <groupId>io.netty</groupId>
             <artifactId>netty-resolver-dns</artifactId>
```

Comment:
I got an OK from Redned, so just one more thing. `netty-resolver-dns`'s version here is 4.1.43 but I did see you bumped PacketLib's Netty version. Will this be an issue?

Reply:
In ideal, all dependencies should be kept in sync to avoid potential issues (mentioned one on Discord)